kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportMapper.kt

```diff
@@ -331,7 +332,8 @@ private fun ObjCExportMapper.bridgeMethodImpl(descriptor: FunctionDescriptor): M
     val returnBridge = bridgeReturnType(descriptor, convertExceptionsToErrors)
 
     if (descriptor.isSuspend) {
-        valueParameters += MethodBridgeValueParameter.SuspendCompletion
+        val useUnitCompletion = (unitSuspendFunctionExport == UnitSuspendFunctionObjCExport.PROPER) && (descriptor.returnType!!.isUnit())
```

Comment:
Is it safe to `!! here?

Reply:
Yes. `bridgeReturnType` above does the same.